src/GuestConfiguration/GuestConfiguration.Test/ScenarioTests/GetVmGuestPolicyStatusHistoryTests.ps1

```diff
@@ -41,6 +55,21 @@ function Get-AzVMGuestPolicyStatusHistory-InitiativeIdScope
 	Assert-True { $historicalStatuses.Count -gt 0 }
 }
 
+<#
+.SYNOPSIS
+Get guest configuration policy status history by Initiative id scope
+#>
+function Get-AzVMGuestPolicyStatusHistory-InitiativeIdScope_Custom
+{
+	$rgName = "amits-test"
+	$vmName = "amits-winser6"
+	$initiativeId = "/subscriptions/b5e4748c-f69a-467c-8749-e2f9c8cd3db0/providers/Microsoft.Authorization/policySetDefinitions/8174929b-2752-46af-9167-89d2d6780439"
```

Comment:
@aashish1987 in order to allow other users to record these tests if needed, we shouldn't hard-code resource ids, as it's not guaranteed the user recording the test has access to this subscription or definition. Tests should properly construct all necessary resources and objects that will be used during the test, and also delete them at the end of every test.

Reply:
@corm@microsoft.com - For Guest Configuration, i did follow what was being used in past.
I totally agree we should not hard code resources as it might result in access issues for others to record the tests.
For now I will mark this as wont fix. 
I will take this change up in a new PR.

---
In reply to: [261284886](https://github.com/Azure/azure-powershell/pull/8574#discussion_r261284886) [](ancestors = 261284886)